pyproject.toml

```diff
@@ -34,7 +34,7 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "jupyter_client>=6.1.3,<8",
+    "jupyter_client>=6.1.3,<=7.4.2",
```

Comment:
```suggestion
    "jupyter_client>=6.1.3,<=7.4.1",
```

Reply:
7.4.2 seems to be ok? Tests are passing

Reply:
Nope. Some tests have been hanging for more than 1h now. Last time I checked, only `7.4.1` works

Reply:
You're right! Thanks